Updated AWS EC2 Documentation (#1158)
kestra-io/docs: content/docs/02.installation/08.aws-ec2.md

4. Create a key-pair to securely connect to your instance. This key is needed to connect through SSH in the following steps.
5. Create a security group that allows SSH traffic from your IP. Also allow HTTPS traffic.

![ec2 creation](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup1.png)

![ec2 key pair](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup2.png)

![ec2 network subgroup](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_setup3.png)

> 

Reviewer: you can change the images and keep the same path

the ../../../ are not needed and lead to those images not rendering 

![image](https://github.com/kestra-io/docs/assets/86264395/7feb7a3e-c8c8-4292-9338-0dc88da5b903)

Download the official Docker-Compose file: `curl -o docker-compose.yml https://raw.githubusercontent.com/kestra-io/kestra/develop/docker-compose.yml`

> Use editor such as Vim to modify the `docker-compose.yml` and set basic authentication to true, and configure your basic authentication credentials to secure your Kestra instance.<br>
> <b>Note:</b> You will need to provide a valid email address for the username

Reviewer: ```suggestion
Use an editor such as Vim to modify the `docker-compose.yml` and set basic authentication to true, and configure your basic authentication credentials to secure your Kestra instance. Make sure to add a valid email address.
```

::

> ![ec2 inbound rules](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_security_group_port_inbound_rules.png)

Reviewer: ```suggestion
![ec2 inbound rules](/docs/administrator-guide/deployment/aws-ec2/ec2_security_group_port_inbound_rules.png)
```

the same applies to all images, you don't need the ../../ — public is used as root for rendering images so you can reference all images starting from that path e.g. /docs/your_folder/your_image.png
